source/Posets/Delta-V-CompletePoset.lagda

```diff
@@ -0,0 +1,854 @@
+
+ ⊑-is-antisymmetric : {P Q : Ω¬¬ 𝓥} → P ⊑ Q → Q ⊑ P → P ＝ Q 
+ ⊑-is-antisymmetric {P} {Q} o r =
+   to-subtype-＝ (λ X → being-¬¬-stable-is-prop fe (holds-is-prop X))
+                 (to-subtype-＝ (λ _ → being-prop-is-prop fe)
+                                (pe (holds'-is-prop P) (holds'-is-prop Q) o r))
+
+ ⊑-is-transitive : (P Q R : Ω¬¬ 𝓥) → P ⊑ Q → Q ⊑ R → P ⊑ R
+ ⊑-is-transitive P Q R o r p = r (o p)
+
+ Ω¬¬-Poset : Poset (𝓥 ⁺) 𝓥
+ Ω¬¬-Poset = (Ω¬¬ 𝓥 ,
+              (λ P → λ Q → (P ⊑ Q , ⊑-is-prop-valued P Q)) ,
+              (⊑-is-reflexive , ⊑-is-transitive) , ⊑-is-antisymmetric)
+
+ open Local-Smallness (𝓥 ⁺)
+                      𝓥
+                      𝓥
+                      Ω¬¬-Poset
+                      (λ P → λ Q → (P ⊑ Q , ⊑-is-prop-valued P Q))
+
+ ⊑-is-locally-small : is-locally-small-order
+ ⊑-is-locally-small P Q = (P ⊑ Q , ≃-refl (P ⊑ Q))
+
+ open δ-complete-poset 𝓥 Ω¬¬-Poset
+
+ Ω¬¬-δ-complete : is-δ-complete
+ Ω¬¬-δ-complete Q R Q-⊑-R P =
+   (((¬¬ (((Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i) holds') holds)) ,
+    negations-are-props fe) ,
+    ¬-is-¬¬-stable) ,
+    (is-upbnd , has-lub-cond)) 
+  where
+   open Joins (λ Q → λ R → (Q ⊑ R , ⊑-is-prop-valued Q R))
+   open propositional-truncations-exist pt
+
+   E : Ω¬¬ 𝓥
+   E = ((¬¬ ((Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i) holds') holds) ,
+        negations-are-props fe) ,
+        ¬-is-¬¬-stable)
+   
+   is-upbnd : (E is-an-upper-bound-of (δ-fam Q R P)) holds 
+   is-upbnd i δ-i not-exists = not-exists ∣ (i , δ-i) ∣
+
+   has-lub-cond : ((U , _) : upper-bound (δ-fam Q R P)) → E ⊑ U
+   has-lub-cond (U , U-is-upbnd) = E-⊑-U
+    where
+     untrunc-map : Σ i ꞉ (𝟙 + (P holds)) , δ Q R P i holds' → U holds'
+     untrunc-map (i , δ-i) = U-is-upbnd i δ-i
+     f : (Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i) holds') holds → U holds'
+     f = ∥∥-rec (holds'-is-prop U) untrunc-map
+     g : ¬¬ ((Ǝ i ꞉ (𝟙 + P holds) , (δ Q R P i) holds') holds) → ¬¬ (U holds')
+     g = double-contrapositive f
+     h : ¬¬ (U holds') → U holds'
+     h = holds'-is-¬¬-stable U
+     E-⊑-U : E ⊑ U
+     E-⊑-U = h ∘ g
+
+ open non-trivial-posets Ω¬¬-Poset
+
+ Ω¬¬-is-non-trivial : is-non-trivial-poset
+ Ω¬¬-is-non-trivial = ((⊥ , 𝟘-is-¬¬-stable) , (⊤ , 𝟙-is-¬¬-stable) ,
+                      𝟘-elim , (λ np → 𝟘-is-not-𝟙 (ap (pr₁ ∘ pr₁) np)))
+
+\end{code}
+
+Now we can prove Theorem 6.2.21 from Tom de Jong's thesis.
+
+\begin{code}
+
+module Predicative-Taboos (𝓤 𝓦 𝓥 : Universe) (A : Poset 𝓤 𝓦) where
+
+ open δ-complete-poset 𝓥 A
+ open non-trivial-posets A
+ open Positive-Posets 𝓤 𝓦 𝓥 A
+ open positive-posets
+ open Local-Smallness 𝓤 𝓦 𝓥 A _≤_
+ open Small-δ-complete-poset 𝓤 𝓦 𝓥 A
+ open Retract-Lemmas 𝓤 𝓦 𝓥 A
+
+ small-non-trivial-poset-implies-¬¬resizing :
+   (δ-complete : is-δ-complete)
+  → is-non-trivial-poset
+  → small-δ-complete-poset.poset-is-small δ-complete
+  → Ω¬¬-Resizing 𝓥 𝓥
+ small-non-trivial-poset-implies-¬¬resizing δ-complete (x , y , x-≤-y , x-≠-y) (locally-small , carrier-small) =
```

Comment:
You can do
```
small-non-trivial-poset-implies-¬¬resizing δ-complete 
 (x , y , x-≤-y , x-≠-y) (locally-small , carrier-small) =
```
to fix the width.
Also, please rename `x-≤-y` and `x-≠-y` as mentioned before.